mw-webapp/src/component/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,64 @@
+import {useState} from "react";
+import clsx from "clsx";
+import styles from "src/component/checkbox/Checkbox.module.scss";
+
+/**
+ * Checkbox props
+ * {@link Checkbox}
+ */
+interface CheckboxProps {
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Default checkbox status
+   * @default false
+   */
+  isDefaultChecked?: boolean;
+  /**
+   * Checkbox`s value
+   */
+  value?: string;
+  /**
+   * Checkbox's name
+   */
+  name?: string;
+  /**
+   * The checkbox must be filled out if true
+   * @default false
+   */
+  required?: boolean;
+  /**
+   * Checkbox's class
+   */
+  className?: string | string[];
+}
+
+/**
+ * Checkbox component
+ */
+export const Checkbox = (props : CheckboxProps) => {
+  const [isChecked, setIsChecked] = useState(!!props.isDefaultChecked);
+
+  /**
+   * Event handler for the checkbox change event
+   */
+  const onChangeHandler = () => (
```

Comment:
1. It does not handle any events
2. It is not required to comment private functions

Reply:
2. According our eslint rules jsdoc required to all arrow functions too (not only public)

Reply:
moved the function inside onChange

Reply:
how to use this component now?